code/unit_tests/organ_tests.dm

```diff
@@ -251,4 +251,141 @@
 	else
 		pass("All organs were removed and replaced correctly.")
 
-	return 1
\ No newline at end of file
+	return 1
+
+// ==============================================================================
+// Stumps shall not drop
+// ==============================================================================
+/datum/unit_test/stumps_shall_not_drop
+	name = "ORGAN: Stumps Shall Not Drop From a Gibbed Mob or Severed Limbs."
+
+/datum/unit_test/stumps_shall_not_drop/proc/find_stumps()
+	//Look for stumps that aren't deleted
+	var/list/found_stumps = (locate(/obj/item/organ/external/stump) in world)
+	for(var/obj/item/organ/external/stump/O in found_stumps)
```

Comment:
```suggestion
	for(var/obj/item/organ/external/stump/O in world)
```

Reply:
(Had to commit it in two part since apparently I didn't add it to the batch properly.)